### PR TITLE
Add ToomanPay interview experience by SirSaleh

### PR DESCRIPTION
**Pull request by SirSaleh**
Please merge my commits into your repo Sotoon.

# Changes

* A New directory for `TomanPay`;
* A log of questions they asked me in the interview and my opinion about the interview process.
* A link to the the document in main `README.md`.

### CODE REVIEW

File: src/TomanPay/senior_backend_position.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## Senior Backend Developer (Python/Django)
+### Interview process
+```mermaid
+flowchart LR
+    sr(Send resume) --> hr(HR call) --> ti1(1st Technical Interview) --rejected--x  ti2(2nd Technical Interview) -.-> hri(HR Interview) -.-> o(Offer)
+```
+### Apply way
+Refer
+
+### Interview date
```
Comment (mo1ein) [inline src/TomanPay/senior_backend_position.md:14]: `Interview Date` is better.

File: src/TomanPay/senior_backend_position.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- **Refered** <br /> Unknown!
```
Comment (mo1ein) [inline src/TomanPay/senior_backend_position.md:15]: We don't need this line. Delete it.

File: src/TomanPay/senior_backend_position.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+- **HR Call**<br /> 1403.03.27
+
+- **Technical Interview** <br> 1403.03.30
+
+- **Rejection Email** <br /> 1400.07.26
+
+### Interview duration
```
Comment (mo1ein) [inline src/TomanPay/senior_backend_position.md:23]: `Interview Duration` is better.

File: src/TomanPay/senior_backend_position.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- **Technical Interview** <br> 1 Hour and 30 minutes
+
```
Comment (mo1ein) [inline src/TomanPay/senior_backend_position.md:25]: Add here your `Interview Platform` like this: 
```
### Interview Platform
Microsoft Teams
```

File: src/TomanPay/senior_backend_position.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+### 1st Technical Interview
+
+- Interviewer Told about himself and The culture in Toman.
+
+- Q: Tell me about yourself and your experience.
```
Comment (mo1ein) [inline src/TomanPay/senior_backend_position.md:30]: No need `Q:`, remove it from all lines.

**Review by mo1ein — COMMENTED** [review]
Thanks for the contribution!
Please fix comments.

File: src/TomanPay/senior_backend_position.md (hunk 2 above)
Comment (SirSaleh) [reply src/TomanPay/senior_backend_position.md:15]: use please after "it". We need it. thanks.

Comment (mo1ein) [reply src/TomanPay/senior_backend_position.md]: =))

**Comment by SirSaleh** [comment]
Done. Check it out please! @mo1ein

File: README.md [hunk 6]
```diff
@@ -62,3 +62,10 @@
 <p dir="rtl">
 مصاحبه‌ها بر اساس ترتیب زمانی از جدید به قدیم مرتب شده‌اند.
 </p>
+
+<p dir="rtl">
+مصاحبه‌های دیگران:
+</p>
+
+- [Tooman Pay](src/TomanPay/senior_backend_position.md)
```
Comment (mo1ein) [inline README.md:70]: Please Change this line to:
```
- [Toman Pay](src/TomanPay/senior_backend_position.md)
```
Also if you rename `TomanPay` to `tomanpay`, that makes me happy!

**Review by mo1ein — COMMENTED** [review]
You should add your `README.md` changes to `src/README.md` too.

**Comment by SirSaleh** [comment]
Be Happy.

**Comment by SirSaleh** [comment]
Be happy more.

**Comment by mo1ein** [comment]
LGTM! Thanks man. :)

**Pull request merged**